src/Blocks/continuous.jl

```diff
@@ -20,7 +20,7 @@ Initial value of integrator state ``x`` can be set with `x`
 @mtkmodel Integrator begin
     @extend u, y = siso = SISO()
     @variables begin
-        x(t) = 0.0, [description = "State of Integrator"]
+        x(t), [guess = 0.0, description = "State of Integrator"]
```

Comment:
> Integrators always start at zero though?

Most of the time this is what users want, but it is occasionally useful to have the system solve for an integrator value that makes an initial control signal have a particular value to avoid initial transients, so forcing the user to specify the initial state explicitly might not be too bad

Reply:
I guess the user can not easily _remove_ a default value that has been specified in the library? If they cannot remove the value, we must not provide it since then it cannot be solved for

Reply:
So the explicit state variables, like those in `Integrator, Derivative, FirstOrder, SecondOrder` should keep their defaults, but `y,u` in SISO etc. should not

Reply:
Yes, I am probably going to make a new PR, since almost everything in this is not correct :(

Reply:
@ChrisRackauckas We can probably close this without merging since this PR has more incorrect than correct things, I'll make a new PR for the things that I would like to fix and the docs.